geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/AbstractIngest.scala

```diff
@@ -212,13 +214,12 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
   }
 
   protected def runDistributed(): Unit = {
-    beforeRunTasks()
-    Command.user.info("Running ingestion in distributed mode")
```

Comment:
was there a reason to move these two lines out of this method? I'd rather keep them here for consistency with `runLocal()`

Reply:
Yes, because runDistributed gets overwritten in ConverterIngest.